.goreleaser.yml

```diff
@@ -0,0 +1,78 @@
+project_name: helm
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+      - arm
+      - "386"
+      - ppc64le
+      - s390x
+      - riscv64
+    goarm:
+      - "7"
+    ignore:
+      - goos: darwin
+        goarch: "386"
+      - goos: darwin
+        goarch: arm
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+      - goos: darwin
+        goarch: riscv64
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: ppc64le
+      - goos: windows
+        goarch: s390x
+      - goos: windows
+        goarch: riscv64
+    main: ./cmd/helm
+    binary: helm
+    ldflags:
+      - -w -s
+      - -extldflags "-static"
```

Comment:
There are a bunch of (important) LDFLAGs missing here:

https://github.com/helm/helm/blob/main/Makefile#L56-L71

Note: https://github.com/helm/helm/pull/31296 (and a followup PR based on https://github.com/helm/helm/pull/13180/files#diff-8aa632a6f4ad2a3a7fef52661816aa3977f23c0673ef29633c01116fee6745c6R88-R99 would remove the need for most (all?) of these)

Reply:
Notably, `helm version` prints empty git details (and the Kubernetes client-go versions would be unset):

```
$ dist/helm_darwin_arm64/helm version
version.BuildInfo{Version:"v4.0", GitCommit:"", GitTreeState:"", GoVersion:"go1.25.1"}
```

Reply:
```
% ./_dist/helm_darwin_arm64/helm version
version.BuildInfo{Version:"v4.0+unreleased", GitCommit:"c01ac8bb28dcaffe3f0411b280331288123cada9", GitTreeState:"dirty", GoVersion:"go1.25.1"}

```